maps/randomvaults/black_site_prism.dmm

```diff
@@ -51,7 +51,7 @@
 "de" = (/obj/machinery/light{color = "#FF0000"; light_color = "#FF0000"},/turf/simulated/floor/carpet,/area/vault/black_site_prism)
 "dn" = (/obj/structure/disposalpipe/segment{dir = 4},/turf/simulated/wall/invulnerable,/area/vault/black_site_prism)
 "dF" = (/obj/structure/grille,/obj/structure/window/reinforced{dir = 8},/obj/structure/window/reinforced{dir = 4; one_way = 1},/obj/structure/window/reinforced{dir = 1},/obj/structure/cable/yellow{d1 = 1; d2 = 2; icon_state = "1-2"},/turf/unsimulated/floor{icon_state = "dark"},/area/vault/black_site_prism)
-"dG" = (/obj/structure/table,/obj/item/weapon/reagent_containers/glass/beaker{pixel_x = 5},/obj/item/weapon/storage/box/donkpockets{pixel_x = 3; pixel_y = 3},/obj/item/weapon/kitchen/utensil/knife/large,/obj/abstract/map/spawner/voxfood/turf/simulated/floor{icon_state = "white"},/area/vault/black_site_prism)
+"dG" = (/obj/structure/table,/obj/item/weapon/reagent_containers/glass/beaker{pixel_x = 5},/obj/item/weapon/storage/box/donkpockets{pixel_x = 3; pixel_y = 3},/obj/item/weapon/kitchen/utensil/knife/large,/obj/abstract/map/spawner/voxfood,/turf/simulated/floor{icon_state = "white"},/area/vault/black_site_prism)
```

Comment:
Wow, how did this one even happen?

Reply:
it was me, find-and-replacing stuff

Reply:
Is it supposed to say "voxfood," or "table,"?